apis/kafka/generator-config.yaml

```diff
@@ -18,9 +18,3 @@ resources:
       errors:
         404:
           code: NotFoundException
-    fields:
-      ClusterPolicyVersion:
-        is_read_only: true
-        from:
-          operation: GetClusterPolicy
-          path: CurrentVersion
```

Comment:
Why did you removed these lines?

Reply:
Sorry about that, some code left when I made the rebase. Should be solved on my last commit. Thanks for reviewing.